src/main/resources/logback.xml

```diff
@@ -30,7 +30,7 @@
         </encoder>
     </appender>
 
-    <root level="info">
+    <root level="infoSecvice">
```

Comment:
Why do you need this?

Reply:
I don't know how this happened.
It was accidentally renamed.